fiftyone/operators/types.py

```diff
@@ -610,6 +610,23 @@ def tuple(self, name, *items, **kwargs):
         self.define_property(name, tuple_type, view=tuple_view, **kwargs)
         return tuple_type
 
+    def tree(self, name, *items, **kwargs):
+        """Defines a tree property on the object.
+
+        Args:
+            name: the name of the property
+            *items: the types of the items in the tree
+
```

Comment:
Can we add an example here on how to use `tree` as well? @lanzhenw 

It can either be here or at the `TreeSelectionView` level

Reply:
I added at the TreeSelectionView level